### DISCUSSION TITLE
Facing DB connection issue while building Dockerfile using directus docker image

### DISCUSSION THREAD

**Discussion by SaranyaPonnusamy** [directus/directus#16879] [Archive]
Dockerfile:

FROM directus/directus:latest
ENV PUBLIC_URL = "http://0.0.0.0:8055/"
ENV DB_CLIENT = 'pg'
ENV DB_HOST ="localhost"
ENV DB_PORT = "5432"
ENV DB_DATABASE = "directus"
ENV DB_USER = "postgres"
ENV DB_PASSWORD ="postgres"
ENV DB_SSL = "false"
ENV KEY ="key"
ENV SECRET ="secret"
CMD ["yarn", "directus", "start"]

Facing some issue while connecting to database . It throws below error

Error:
Error: knex: Unknown configuration option 'client' value = pg. Note that it is case-sensitive, check documentation for supported values.

Can some one help me with this issue?

**Comment by rijkvanzanten** [comment]
I think it's just a typo in your docker file. AFAIK there's no spaces around the `=` and no quotes around the value. Try it with `ENV DB_CLIENT=pg` (and make sure to update the others)
